src/recipes/editable.ts

```diff
@@ -10,7 +10,9 @@ const editableRecipe = defineRecipe({
     /* Cannot use padding-bottom on editable, as clicking it causes the selection to go to the end. */
     padding: '0 1em 0 0.333em',
     boxSizing: 'border-box',
-    margin: '-0.5px 0 0 calc(1em - 18px)',
+    // Adjust the space between the bullet and the text with margin-left. When font size is greater than 18px, the space will be increased. When the font size is less than 18th, the space will be decreased.
+    // The margin-right must offset margin-left to ensure that urls are not incorrectly ellipsized on smaller font sizes. This is because text-overflow: 'ellipsis' applied to the ThoughtAnnotation does not account for the negative margin of the child.
+    margin: '-0.5px calc(18px - 1em) 0 calc(1em - 18px)',
```

Comment:
Please take note of this comment. We need good documentation for tricky styles like this. This will ensure that future developers will have everything they need to understand why it's there.